Gems/OpenXRVk/Code/Source/OpenXRVkSwapChain.cpp

```diff
@@ -158,10 +160,20 @@ namespace OpenXRVk
                 AZ_Printf("OpenXRVk", "Swapchain Formats: %s\n", swapchainFormatsString.c_str());
             }
 
+            double xrViewResolutionScale = 1.0;
+            if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get())
+            {
+                AZ::SettingsRegistryMergeUtils::PlatformGet(*settingsRegistry, xrViewResolutionScale,
+                    "/O3DE/Atom", "OpenXRViewResolutionScale");
```

Comment:
Since this is a new SR key for OpenXR, maybe we can organize it better.  For instance:
"/O3DE/Atom/OpenXR/ViewResolutionScale"

That way additional OpenXR render settings have a path they can be put under.

Reply:
Good suggestion. I'll change "/O3DE/Atom/OpenXREnable" to "/O3DE/Atom/OpenXR/Enable" then so they are both consistent.

Reply:
I forgot about this "OpenXREnable" setting, does changing this have any effect on the `-openxr=enable` command-line arg that we've been using?

Reply:
>I forgot about this "OpenXREnable" setting, does changing this have any effect on the -openxr=enable command-line arg that we've been using?

No, you can keep using the command line option like that, but if you want to use the setting registry instead you can have a .setreg file with:

````
    "O3DE": {
        "Atom": {
            "OpenXR": {
                "Enable": true
            }
        }
    }
````